### DISCUSSION TITLE
The plugin not works

### DISCUSSION THREAD

**Discussion by Ninokeep** [kylechui/nvim-surround#203] [Q&A] [answered]
### Hello everybody,  

i'm beginner in vim and neovim.
I set up my environnement of neovim and i installed the plugin nvim-surround 

My configuration 
```
NVIM v0.8.1
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by brew@Ventura-arm64.local
```
in my packer.lua i writed this  :
```
use {
'kylechui/nvim-surround',
tag = "*", -- Use for stability; omit to use `main` branch for the latest features
config = function()
require('nvim-surround').setup({})
end
}
```

but he doesn't work when i'm on string and i write ysiw)  he doesn't insert () :-/

**Comment by kylechui** [comment, marked as answer]
Did you `require("packer")` from your `init.lua`? Are you typing `ysiw)` in normal mode or insert mode (it should be in normal mode)?

**Reply by Ninokeep** [reply]
Hello kylechui !
My config looks like :
config
|--after
|-----plugin
|----------- treesitter.lua
|----------- etc..
|--lua
|---MyNameProfile
|---- init.lua
|---- packer.lua
|---- remap.lua
|--init.lua

All plugins work correctly but the command ysiw) in normal mode not works.
I follow the configuration in your readme

**Reply by kylechui** [reply]
Did you do `:PackerSync`, and if you have, please send the output for `:verbose nmap ys`.

**Reply by Ninokeep** [reply]
n  yss           <Lua 694: ~/.local/share/nvim/site/pack/packer/start/nvim-surround/lua/nvim-surround/config.lua:524>
                 Add a surrounding pair around the current line (normal mode).
n  ys          * <Lua 724: ~/.local/share/nvim/site/pack/packer/start/nvim-surround/lua/nvim-surround/init.lua:74>
                 Add a surrounding pair around a motion (normal mode).

**Reply by Ninokeep** [reply]
@kylechui  now he works !! thanks but why the command :PackerSync  he works ?

**Reply by kylechui** [reply]
The short version is that putting the `use ...` code into your configuration doesn't actually *install* my plugin onto your system locally. `:PackerInstall` installs plugins while `:PackerSync` syncs your installed plugins with your configuration file (including installing them). You can read more about it in Packer's help menu
